modules/keycode/keycode.js

```diff
@@ -1,5 +1,6 @@
 /**
  * Enumerates the supported keys.
+ * NOTE: The maps represents actual keys on the keyboard not chars. 
```

Comment:
Just leaving a comment here that we've discussed it offline and this is Jitsi Meet API for the remote control. The value are the same as in RobotJS, but only for convenience.